code/modules/surgery/organs/eyes.dm

```diff
@@ -443,10 +443,12 @@
 /obj/item/organ/internal/eyes/robotic/glow/on_insert(mob/living/carbon/eye_owner)
 	. = ..()
 	RegisterSignal(eye_owner, COMSIG_ATOM_DIR_CHANGE, PROC_REF(update_visuals))
+	RegisterSignal(eye_owner, COMSIG_MOVABLE_MOVED, PROC_REF(update_visuals))
 
 /obj/item/organ/internal/eyes/robotic/glow/on_remove(mob/living/carbon/eye_owner)
 	. = ..()
 	UnregisterSignal(eye_owner, COMSIG_ATOM_DIR_CHANGE)
+	UnregisterSignal(eye_owner, COMSIG_MOVABLE_MOVED)
```

Comment:
```suggestion
	UnregisterSignal(eye_owner, list(COMSIG_ATOM_DIR_CHANGE, COMSIG_MOVABLE_MOVED))
```

`UnregisterSignal()` allows lists of multiple signals, so may as well condense this easy-peasy.